api/src/main/java/org/apache/cloudstack/api/command/user/vm/UpdateVmNicIpMtuCmd.java

```diff
@@ -46,9 +28,27 @@
 import com.cloud.uservm.UserVm;
 import com.cloud.utils.net.NetUtils;
 import com.cloud.vm.Nic;
+import com.google.common.base.Strings;
+import org.apache.cloudstack.api.APICommand;
+import org.apache.cloudstack.api.ApiArgValidator;
+import org.apache.cloudstack.api.ApiCommandJobType;
+import org.apache.cloudstack.api.ApiConstants;
+import org.apache.cloudstack.api.ApiConstants.VMDetails;
+import org.apache.cloudstack.api.ApiErrorCode;
+import org.apache.cloudstack.api.BaseAsyncCmd;
+import org.apache.cloudstack.api.Parameter;
+import org.apache.cloudstack.api.ResponseObject.ResponseView;
+import org.apache.cloudstack.api.ServerApiException;
+import org.apache.cloudstack.api.response.NicResponse;
+import org.apache.cloudstack.api.response.UserVmResponse;
+import org.apache.cloudstack.context.CallContext;
+import org.apache.log4j.Logger;
 
-@APICommand(name = "updateVmNicIp", description = "Update the default Ip of a VM Nic", responseObject = UserVmResponse.class)
-public class UpdateVmNicIpCmd extends BaseAsyncCmd {
+import java.util.ArrayList;
+import java.util.EnumSet;
+
+@APICommand(name = "updateVmNicIpMtu", description = "Update the default Ip and MTU of a VM Nic", responseObject = UserVmResponse.class)
```

Comment:
rename of a public API is a big no-no. it breaks backwards compatibility. if the extra parameter is optional, you can just add it, else create an extra API, please.